### DISCUSSION TITLE
Arduino Library Release v2.4.0

### DISCUSSION THREAD

**Discussion by Makuna** [Makuna/Rtc#178] [Announcements]
RtcDateTime has been enhanced:

- support methods to retrieve and set Unix time values.  
- support methods to retrieve and set Ntp time values.  
- a new InitWithDateTimeFormatString() method that takes a user provided format string to convert a string to RtcDateTime

**Comment by nickemp** [comment]
Hi Can I Do alarms with this and how? 
sorry very new to RTCds1302 and coding

**Reply by Makuna** [reply]
@nickemp The DS1302 chip doesn't have alarms; so, I can't support something that isn't there.  You could always look for a timer library to give you that support if you don't want to use another RTC.  DS3231 & DS3234 both have alarm capability and are supported.

**Reply by Makuna** [reply]
@nickemp latest release now has an alarm manager that doesn't rely on a feature of the RTC module.

**Comment by nickemp** [comment]


**Reply by Makuna** [reply]
This topic that you're posting under is about the Arduino Library Release announcement.  You should go to the Q&A area and create a new topic on your questions.
